app-typescript/components/DateTimePicker.tsx

```diff
@@ -173,7 +173,8 @@ export class DateTimePicker extends React.PureComponent<IProps> {
                             data-test-id="date-input"
                         />
                     </div>
-                    <div style={{flexGrow: 1}}>
+                    <div style={{flexGrow: 0, lineHeight: '3.2rem', color: 'var(--color-text-muted)'}}>@</div>
```

Comment:
`line-height` is only used for centering here which stops working if input field height changes. Coupling can be avoided and code stability improved if centering was handled by flex parent. Not sure why is `align-items` is set to `end` at the moment, but setting it to `center` handles vertical centering without introducing coupling.

Is `flexGrow` doing anything? It renders the same without it.

Reply:
The input height will not change, as it has the same hardcoded value. We should replace it with a variable at some point, but this is fine for now. 

Spacer settings where not introduced in this PR or by me. I would say that it is set to `end` because of the possibility of individual labals for date and time; `center` would misalign the '@' and the button at the end.

`flexGrow: 1` on the inputs is necessary. `flexGrow: 0` on the '@' is indeed not and can be removed.